rasa/core/tracker_store.py

```diff
@@ -499,9 +536,8 @@ def retrieve(self, sender_id):
             )
 
         if stored is not None:
-            return DialogueStateTracker.from_dict(
-                sender_id, stored.get("events"), self.domain.slots
-            )
+            events = self._events_since_last_session_start(stored)
+            return DialogueStateTracker.from_dict(sender_id, events, self.domain.slots)
```

Comment:
I thought about this on Sunday and having the model domain in the tracker store is not good imo.

Reply:
yeah but how would you slot the domain slots in the tracker?

Reply:
the agent could do so.

e.g. 

```
events = store.load(sender_id)
self.initialise_tracker_with_model_context(events)
```

Reply:
okay yeah, let's tackle this separately